### ISSUE TITLE
a11y: Color Contrast - About

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13707]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF
Ratio: 3.16:1 
Main Content
- Home, Rancher, Dashboard, Rancher CLI, Helm, Machine, View release notes, Download, rancher-darwin-amd64-v2.9.2.tar.gz, rancher-linux-amd64-v2.9.2.tar.gz and rancher-windows-386-v2.9.2.zip links. (on default state) 

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1
Main Content 
- Diagnostics Button 

Foreground Text Color: #FFFFFF
Background Text Color: #297DB4
Ratio: 4.48:1 
Main Content
- Diagnostics button (on focus) 
        
## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Resolved in VPAT. Issue 244
